### DISCUSSION TITLE
After reviewing we have determined that your account is ineligible for GitHub purchases due to restrictions under U.S. economic sanctions.

### DISCUSSION THREAD

**Discussion by mussyaroslav** [community/community#119648] [GitHub Education]
![Снимок экрана 2024-04-15 123350](https://github.com/community/community/assets/68197667/fb91ce71-d581-4cbe-815e-9cfbe1ed009d)
### Hi everyone,

After receiving approved status on github student developer pack, I received this message.

Thank you for your patience during this process. After reviewing we have determined that your account is ineligible for GitHub purchases due to restrictions under U.S. economic sanctions. For this reason you will not be able to proceed with your GitHub transaction.
If you disagree with this decision, or want to learn more, please go to https://docs.github.com/en/github/site-policy/github-and-trade-controls.
Apologies for any inconvenience.

But after following this link I did not find any restrictions in Russia. It says there that the restrictions apply only to Crimea and Dombas.

**Comment by yeatttttt** [comment]
> ![Снимок экрана 2024-04-15 123350](https://private-user-images.githubusercontent.com/68197667/322391411-fb91ce71-d581-4cbe-815e-9cfbe1ed009d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzODMwNjQsIm5iZiI6MTcyMTM4Mjc2NCwicGF0aCI6Ii82ODE5NzY2Ny8zMjIzOTE0MTEtZmI5MWNlNzEtZDU4MS00Y2JlLTgxNWUtOWNmYmUxZWQwMDlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE5VDA5NTI0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2ZWNhNTQ4MTgyNDVhZWFkMDk2YmNjNmFhNjVlNjRhZDRlNzkwYzRkYmI2YTgyZmE0NTg1YjlkMjY0MWRjOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qxztK6atbcpIy4NWKHuH4ygz2qIT1XnGeZPfTYN7OtQ)
> 
> ### Hi everyone,
> After receiving approved status on github student developer pack, I received this message.
> 
> Thank you for your patience during this process. After reviewing we have determined that your account is ineligible for GitHub purchases due to restrictions under U.S. economic sanctions. For this reason you will not be able to proceed with your GitHub transaction. If you disagree with this decision, or want to learn more, please go to https://docs.github.com/en/github/site-policy/github-and-trade-controls. Apologies for any inconvenience.
> 
> But after following this link I did not find any restrictions in Russia. It says there that the restrictions apply only to Crimea and Dombas.

I have the same problem. Did you get students pack afterwards?

**Reply by twiborg** [reply]
Have you been able to solve this problem?

**Reply by Temmi556** [reply]
Удалось решить проблему?

**Reply by ww249** [reply]
Have you been able to solve this problem?

**Reply by mussyaroslav** [reply]
Да я получил фиолетовый апрув на следующий день, это было в 2024г. В 2025г подписку продлили автоматически.

**Reply by BonyLord** [reply]
Была ли у вас проблема с платёжной информацией?
